pandas/core/strings/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .accessor import StringMethods
+from .base import BaseStringArrayMethods
+
+__all__ = ["StringMethods", "BaseStringArrayMethods"]
```

Comment:
Import here also the other classes that are used outside of pandas.core.strings (`StringArrayMethods`, `CategoricalStringMethods`) ?

Reply:
The others are gone now that we're using a methods mixing & inheritance.